apps/web/src/utils/date.ts

```diff
@@ -32,8 +32,35 @@ export function formatDate(
   return date;
 }
 
-export function formatTtl(ttl: number) {
```

Comment:
I feel like `formatTtl` was more readable and self-explanatory

Reply:
Done [fb307b4](https://github.com/Blobscan/blobscan/pull/503/commits/fb307b4011a4ef098de11c602ad75819b6c11ca8)